serveradmin/servershell/views.py

```diff
@@ -17,7 +17,7 @@
 from django.utils.html import mark_safe, escape as escape_html
 
 from adminapi.base import QueryError
-from adminapi.filters import Any, InsideOnlyNetwork, Startswith, filter_classes
+from adminapi.filters import Any, ContainedOnlyBy, Startswith, filter_classes
```

Comment:
Guess you also checked for all other places which `InsideOnlyNetwork` was used.

Reply:
It was me who invented it and not exposed to the client side.  I am sure nobody else knows about it.